test/e2e/snaps/test-snap-uilinks.spec.js

```diff
@@ -112,12 +129,12 @@ describe('Test Snap UI Links', function () {
         });
 
         // switch to new tab
-        await driver.switchToWindowWithTitle('MetaMask Snaps Directory');
+        await driver.switchToWindowWithTitle('E2E Test Page');
 
         // check that the correct page has been opened
         await driver.waitForSelector({
-          text: 'Most popular',
-          tag: 'h2',
+          testId: 'empty-page-body',
+          text: 'Empty page by MetaMask',
```

Comment:
we change this selector because now instead of going to the live Snaps site, we go to a mocked page